src/drivers/issue.rs

```diff
@@ -0,0 +1,13 @@
+// Struck that represents a common issue interface between Drivers
+pub struct Issue {
+    pub id: String,
```

Comment:
Why `String` and not `&str` of some kind? Isn't `String` a mutable string that we don't care for?

Reply:
(also for the rest)

Reply:
so extracting from the docs:
> use String if you need owned string data (like passing strings to other threads, or building them at runtime), and use &str if you only need a view of a string.

This is generated from a json (or that is the idea so I understood that that is runtime but I can change that is just I still don't use much slices since I'm not familiar with them 😄